packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -3654,7 +3654,9 @@ module('Integration | operator-mode', function (hooks) {
     assert
       .dom('[data-test-card-url-bar-error]')
       .containsText('This resource does not exist');
-    await percySnapshot(assert);
+    // Percy is failing to capture this snapshot for some
+    // reason. creating issue for this CS-6780
+    // await percySnapshot(assert);
```

Comment:
Already tracking this in [CS-6767](https://linear.app/cardstack/issue/CS-6767/investigate-percy-timeouts)